packages/mui-base/src/utils/useLatestRef.ts

```diff
@@ -0,0 +1,14 @@
+'use client';
+import * as React from 'react';
+import { useEnhancedEffect } from './useEnhancedEffect';
+
+/**
+ * @ignore - internal hook.
+ */
+export function useLatestRef<T>(value: T) {
```

Comment:
we don't have it...
![image](https://github.com/mui/base-ui/assets/1423607/42424e00-0691-4755-9d6b-629894a42e28)

Reply:
It's fine to stay here. One issue I have with `mui-utils` is that it's a separate repo, so if a PR needs a change to an util, it can't be done in one step.

Sidenote but I didn't see the hook used in this PR, not sure if I missed it or it's not used.

Reply:
It is used for the `format` prop